tests/scripts/combined-coverage.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 
 npm run test:unit
+CURRENT_NODE_VERSION=`node -v`
+if [[ $CURRENT_NODE_VERSION =~ 17(.*)|18(.*) ]]
+then
+  echo -e "currently using node version 17+ - setting NODE_OPTIONS \n"
+  unset NODE_OPTIONS
```

Comment:
out of curiosity why do we need to unset NODE_OPTIONS? We set NODE_ICU_DATA in https://github.com/yext/answers-hitchhiker-theme/blob/master/package.json#L16 without needing to unset it

Reply:
There was an error `node: --openssl-legacy-provider is not allowed in NODE_OPTIONS` when using node v14 and I tried to resolve it with unset node option. I will remove it.